lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -45,6 +42,11 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="System.Text.Json" Version="8.0.4" />
```

Comment:
Could this also be moved to `netstandard2.0`?
`System.Text.Json` is alread there for `net8.0`.

Reply:
done!